src/api/routes/delegation.ts

```diff
@@ -22,11 +22,55 @@ import * as m from '$lib/paraglide/messages';
 const makeEntryCode = customAlphabet('6789BCDFGHJKLMNPQRTW', 6);
 
 export const delegation = new Elysia()
-	.use(CRUDMaker.getAll('delegation'))
 	.use(CRUDMaker.updateOne('delegation'))
 	.use(CRUDMaker.deleteOne('delegation'))
 	.use(permissionsPlugin)
 	.use(languageExtractor)
+	.get(
+		'delegation',
+		async ({ permissions, query }) => {
+			permissions.mustBeLoggedIn();
+			return await db.delegation.findMany({
+				where: {
+					conferenceId: query.conferenceId,
+					supervisors: {
+						some: { id: query.supervisorId }
+					},
+					AND: [permissions.allowDatabaseAccessTo('list').Delegation]
+				},
+				include: {
+					_count: {
+						select: {
+							members: true,
+							supervisors: true,
+							appliedForRoles: true
+						}
+					}
+				},
+				orderBy: {
+					entryCode: 'asc'
```

Comment:
Why are we sorting for the entry code? Just curious :D Just so we have a fixed order in which they are returned every time?

Reply:
Yes, exactly. I thought to have a visually understandable base sorting would make sense – and since I did not want to display the id because of its length in the admin table, i figured this would be an easy solution ;)

Reply:
I see. While sorting for this particular purpose it might make more sense to use the id nonetheless, since the code might change and therefore will disrupt the order of the return entities. But this is marginale so it's fine this way